### DISCUSSION TITLE
读《王氏之死》 | /posts/sjq/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#602] [Announcements]
https://yuanfan.rbind.io/posts/sjq/

**Comment by botsp** [comment]
等真得摸到以后才确定原来警犬也是狗啊>>>记得和阿木狗讲:"世界就是个草台班子"~

**Reply by earfanfan** [reply]
哈哈哈，这位朋友哇，阿木狗听不懂人话呀，我要怎样才能跟它讲清楚……
